builder/job.go

```diff
@@ -44,6 +44,7 @@ var validCommitCommands = map[string]bool{
 	"workdir":    true,
 }
 
+// Config containers all configs for a build job
```

Comment:
Yeah, I did notice I have this problem, and already think hard as much as I can, still slipped some, should search `git diff` next time :)